tests/integration/dogshell/test_dogshell.py

```diff
@@ -31,54 +32,156 @@ def get_temp_file():
         return (tf.name, tf)
 
 
+@pytest.fixture  # (scope="module")
+def dogshell_config():
+    config = ConfigParser()
+    config.add_section("Connection")
+    config.set("Connection", "apikey", API_KEY)
+    config.set("Connection", "appkey", APP_KEY)
+    config.set("Connection", "api_host", os.environ.get("DATADOG_HOST", "https://api.datadoghq.com"))
+    return config
+
+
+@pytest.fixture  # (scope="module")
+def config_file(tmp_path, dogshell_config):
+    """Generate a config file for the dog shell."""
+    filename = tmp_path / ".test.dog.ini"
+    with filename.open("w") as fp:
+        dogshell_config.write(fp)
+    return str(filename)
+
+
+@pytest.fixture
+def dogshell(capsys, config_file, dog):
+    """Helper function to call the dog shell command."""
+    import click
```

Comment:
If needed I can refactor it to capture `stdout` and `stderr` manually. It's just a pain to keep it PY2/3 compatible.